### PR TITLE
fix(logo/hp): add logo on HP and adapt heading part

### PR DESCRIPTION
**Pull request by marineFabien**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/adeo/design-system--styleguide/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Other information

### CODE REVIEW

File: src/data/icons.json [hunk 1]
```diff
@@ -116,38 +116,38 @@
   {
```
Comment (gael-boyenval) [inline src/data/icons.json]: this file should not be part of the PR

File: src/pages/index.scss [hunk 2]
```diff
@@ -135,33 +135,45 @@
 .hero {
   color: $color-grey-000;
 
+  &__logo {
```
Comment (gael-boyenval) [inline src/pages/index.scss]: you tried everything center your element ;)


`text-align: center;` => will center the text inside the SVG
`display: flex;` => no content, no effect... aside from displaying the image at 100% of the width
`justify-content: center;` => what content are you trying to center ? path ? 
`margin: 0 auto $mu250;` => `margin 0` auto works for blocks or inline block, but for inline elements. 


once you found the thing that works, try to remove the useless code.
this does not work on safari, and I'm surprised that it even work on chrome.

this is one right way to do it : 

`display: block;` => set the SVG as a block element
`width: $mu400;` => 
`height: $mu400;` => define width and height
`margin: 0 auto $mu250;` => center it and apply margin at the bottom

File: src/pages/index.scss [hunk 3]
```diff
@@ -135,33 +135,45 @@
...
+    height: $mu400;
+    fill: $color-grey-000;
+  }
+
   &__title {
-    font-size: $mu500;
-    line-height: $mu500;
+    font-size: $mu400;
+    line-height: $mu400;
     letter-spacing: -0.086875rem;
```
Comment (gael-boyenval) [inline src/pages/index.scss]: because you don't use `em` for your letter spacing, witch would be a relative value from the current font-size, you need to change that if you change the font-size. I would recommend you to use an `em` value.

File: src/pages/index.scss (hunk 2 above)
Comment (marineFabien) [reply src/pages/index.scss]: sorry, I did not rewrite my code, it was a copy / paste and I forgot it: - /
shame on me !
